modules/swagger-codegen/src/main/resources/LoopBackSwift/api.mustache

```diff
@@ -0,0 +1,40 @@
+import LoopBackSwift
+
+{{#operations}}public class {{classname}}: LoopBackAPI<{{baseName}}> {
+
+    public required init(requestor: Requestable) {
```

Comment:
Why `required`? I think we want to be able to provide a custom `BaseURLClosure`, as we are at the `LoopBackAPI` level.